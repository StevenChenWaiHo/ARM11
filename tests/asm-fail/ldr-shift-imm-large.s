ldr r3,[r1,r2, lsl #58]
