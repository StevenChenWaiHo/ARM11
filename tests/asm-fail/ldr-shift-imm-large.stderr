tests/asm-fail/ldr-shift-imm-large.s:1:23: Const too large for a shift const: 58 (max is 31)
